src/components/validations/with-validation.hoc.js

```diff
@@ -278,5 +280,19 @@ const withValidation = (WrappedComponent) => {
   return WithValidation;
 };
 
-export { validationsPropTypes };
+function getValidationType({ hasError, hasWarning, hasInfo }) {
+  let type = '';
+
+  if (hasError) {
```

Comment:
I think this function would befit from multiple returns instead of the `if-else` e.g. 

```
if(hasWarning){
    return 'warning';
}
```

Reply:
Updated in commit 02dd7594e